Projects/App/Sources/UI/Place/PlaceList/View/Views/BannerView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  BannerView.swift
+//  App
+//
+//  Created by 리아 on 2022/10/17.
+//  Copyright © 2022 zesty. All rights reserved.
+//
+
+import SwiftUI
+import UIKit
+import SnapKit
+
+final class BannerCell: UICollectionViewCell {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setView()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    private func setView() {
+        let bannerView = BannerView()
+        let controller = UIHostingController(rootView: bannerView)
+        addSubview(controller.view)
+        controller.view.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+    
+}
+
+struct BannerView: View {
+
+    @State private var selected = 0
+    
+    var body: some View {
+        TabView(selection: $selected) {
+            ForEach(0..<4) { index in
+                Text("\(index)")
+                    .tag(index)
+                    .tabItem {
+                        Image(systemName: (selected == index ? "circle.fill" : "circle"))
+                            .environment(\.symbolVariants, .none)
+                    }
+            }
+        }
+        .onAppear {
+            UIPageControl.appearance().currentPageIndicatorTintColor = .black
+            UIPageControl.appearance().pageIndicatorTintColor = .black
```

Comment:
p2
global set 지양하면 좋을 것 같습니다.
```suggestion
            let pageControl = UIPageControl.appearance(whenContainedInInstancesOf: [BannerCell.self])
            pageControl.currentPageIndicatorTintColor = .black
            pageControl.pageIndicatorTintColor = .black
```

Reply:
저도 작성하면서 찝찝했지만 어떻게 고칠 지 감이 안 잡혀서 지나갔던 건데
이렇게 짚어주시고 알려주셔서 정말 감사드립니다!

Reply:
[f13f560](https://github.com/DeveloperAcademy-POSTECH/MacC-TEAM-ZESTY/pull/33/commits/f13f560198c80ccec3a72a08b281b4eccb6babaf) 반영 완료!